extra-tests/classes/RollupQueryBuilderTests.cls

```diff
@@ -112,12 +131,21 @@ private class RollupQueryBuilderTests {
 
   @IsTest
   static void doesNotAddIsoCodeForMultiCurrencyWhenFieldDoesNotExist() {
-    if (RollupCurrencyInfo.IS_MULTICURRENCY == false) {
+    if (RollupCurrencyInfo.isMultiCurrency() == false) {
       return;
     }
 
     String queryString = RollupQueryBuilder.Current.getQuery(ContactPointAddress.SObjectType, new List<String>(), 'Id', '=');
 
     System.assertEquals(false, queryString.contains(RollupCurrencyInfo.CURRENCY_ISO_CODE_FIELD_NAME));
   }
+
+  @IsTest
+  static void correctlyFormatsAggregateQuery() {
+    String countQuery = RollupQueryBuilder.Current.getQuery(Opportunity.SObjectType, new List<String>{ 'count()' }, 'Id', '=');
+
+    Integer amount = Database.countQuery(countQuery);
```

Comment:
Should this variable be called `count` instead of `amount`? It doesn't seem to be doing anything with amounts here

Reply:
`count` is a reserved word in Apex, but I can definitely rename this to something more intuitive!

Reply:
Oooooh right.... stupid reserved words!